api/popUpInfo/alsB.json

```diff
@@ -0,0 +1,16 @@
+  
+{
+    "restaurant": "Al’s Breakfast",
+    "location": {
+        "latitude": 44.981062, 
+        "longitude": -93.235666 
+    },
+    "info": 
+        {
+            "desc": "Small breakfast place: egg specials, pancakes, waffles",
+            "rating": "Star Rating: 5",
+            "address": "Address:\n413 14th Ave SE \nMinneapolis, MN 55414",
```

Comment:
Leave the formatting to the front end; the JSON is just meant to hold data. So "Address:" and "\n" are unnecessary.